lms/djangoapps/ccx/migrations/0004_change_ccx_coach_to_staff.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+from student.roles import CourseCcxCoachRole, CourseStaffRole
+from instructor.access import allow_access, revoke_access
+
+from ccx_keys.locator import CCXLocator
+from lms.djangoapps.ccx.utils import ccx_course
+
+def change_existing_ccx_coaches_to_staff(apps, schema_editor):
+	"""
+	Modify all coaches of CCX courses so that they have the staff role on the
+	CCX course they coach, but retain the CCX Coach role on the parent course.
+
+	Arguments:
+		apps (Applications): Apps in edX platform.
+		schema_editor (SchemaEditor): For editing database schema (unused)
+
+	"""
+	CustomCourseForEdX = apps.get_model('ccx', 'CustomCourseForEdX')
+	list_ccx = CustomCourseForEdX.objects.all()
+	for ccx in list_ccx:
+		ccx_locator = CCXLocator.from_course_locator(ccx.course_id, unicode(ccx.id))
+		with ccx_course(ccx_locator) as course:
+			ccx.coach.is_authenticated = lambda: True
```

Comment:
Nit: Would be great to add an explanatory comment that says something like "# Indicate that this is not an anonymous user; required for the methods below to work"